packages/components/src/theme/charts.scss

```diff
@@ -180,3 +183,68 @@
 .r-report__files .attachment-card:hover {
     text-decoration: none;
 }
+
+.chart-builder-modal {
+    width: calc(100% - 100px) !important;
```

Comment:
I don't think `!important` is necessary here if you define a separate style above that looks like:

```scss
.modal-dialog.chart-builder-modal {
    width: calc(100% - 100px);
}
```

Reply:
done

Reply:
done